diffgraph/cli.py

```
@@ -25,48 +26,7 @@ def is_git_repo() -> bool:
-
-        for file in files:
-            # Skip hidden files
-            if not file.startswith('.'):
-                full_path = os.path.join(root, file)
-                # Convert to relative path
-                rel_path = os.path.relpath(full_path, os.getcwd())
-                all_files.append(rel_path)
-    return all_files
-
-def get_changed_files() -> List[Dict[str, str]]:
+def get_changed_files(diff_args: List[str] = []) -> List[Dict[str, str]]:
```

Ruff (0.12.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)